app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/util/VaultItemListingDataExtensions.kt

```diff
@@ -287,32 +297,48 @@ private fun List<SendView>.toDisplayItemList(
         )
     }
 
+@Suppress("LongParameterList")
 private fun CipherView.toDisplayItem(
     baseIconUrl: String,
     hasMasterPassword: Boolean,
     isIconLoadingDisabled: Boolean,
     isAutofill: Boolean,
     isFido2Creation: Boolean,
+    fido2CredentialAutofillView: Fido2CredentialAutofillView?,
 ): VaultItemListingState.DisplayItem =
     VaultItemListingState.DisplayItem(
         id = id.orEmpty(),
         title = name,
         titleTestTag = "CipherNameLabel",
-        subtitle = subtitle,
+        subtitle = this.toSubtitle(fido2CredentialRpId = fido2CredentialAutofillView?.rpId),
         subtitleTestTag = "CipherSubTitleLabel",
         iconData = this.toIconData(
             baseIconUrl = baseIconUrl,
             isIconLoadingDisabled = isIconLoadingDisabled,
+            usePasskeyDefaultIcon = (isAutofill || isFido2Creation) &&
+                this.isActiveWithFido2Credentials,
         ),
-        iconTestTag = toIconTestTag(),
-        extraIconList = toLabelIcons(),
-        overflowOptions = toOverflowActions(hasMasterPassword = hasMasterPassword),
+        iconTestTag = this.toIconTestTag(),
+        extraIconList = this.toLabelIcons(),
+        overflowOptions = this.toOverflowActions(hasMasterPassword = hasMasterPassword),
         optionsTestTag = "CipherOptionsButton",
         isAutofill = isAutofill,
         isFido2Creation = isFido2Creation,
         shouldShowMasterPasswordReprompt = reprompt == CipherRepromptType.PASSWORD,
     )
 
+private fun CipherView.toSubtitle(fido2CredentialRpId: String?): String? =
+    fido2CredentialRpId
+        ?.takeIf { this.type == CipherType.LOGIN && it.isNotEmpty() && it != this.name }
+        ?.let {
+            if (subtitle.isNullOrEmpty()) {
+                it
+            } else {
+                it + "\n" + subtitle
```

Comment:
Would it make more sense to just add a new nullable property and display it in the composable when present. Then we can avoid the string concatenation stuff here.

Reply:
The Figma designs showed the subtitle as a single multi-line text box so I thought that was what we wanted. I can ask about it!

Reply:
Got the update from the team and changed the text field and test tag accordingly! 👌